### PR TITLE
internal/pkg/scaffold/olm-catalog: allow multiple RBAC files in CSV config, add operator-name field

### PR DESCRIPTION
**Pull request by estroz**
**Description of the change:**
* change CSV config field `role-path -> role-paths` to add multiple RBAC manifests
to a CSV
* add CSV config field `operator-name` (see #1571)
* update documentation

**Motivation for the change:** users can currently add both `Role`s and `ClusterRole`s to their CSV, but they must be in the same file. `role-paths` allows, for example, a `deploy/role.yaml` and `deploy/cluster_role.yaml` to be read and added to a CSV.

See #1571 for `operator-name` details.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ### Changed
 
+- CSV config field `role-path` is now `role-paths` and takes a list of strings. Users can now specify multiple `Role` and `ClusterRole` manifests using `role-paths`. ([#1704](https://github.com/operator-framework/operator-sdk/pull/1704))
```
Comment (hasbro17) [inline CHANGELOG.md]: Does this break `operator-sdk gen-csv ...` for existing CSV configs in projects that define a singular string field?
```YAML
role-path: "deploy/role.yaml"
```

If so then we should label this as a breaking change. Even if it is a small one.

File: doc/user/olm-catalog/generating-a-csv.md [hunk 2]
```diff
@@ -17,23 +17,34 @@ Operator SDK projects have an expected [project layout][doc-project-layout]. In
...
+crd-cr-paths:
+- deploy/crds
+operator-path: deploy/operator.yaml
+role-paths:
+- deploy/role.yaml
+```
+
+Explanation of all config fields:
+
+- `crd-cr-paths`: list of strings - a list of CRD and CR manifest file/directory paths. Defaults to `[deploy/crds]`.
```
Comment (hasbro17) [inline doc/user/olm-catalog/generating-a-csv.md]: Just noticed this but what do the square brackets [ ] signify?
Line 33 below doesn't have them: `Defaults to deploy/operator.yaml`

Comment (hasbro17) [reply doc/user/olm-catalog/generating-a-csv.md]: Ah never mind, it's a list.

Comment (estroz) [reply doc/user/olm-catalog/generating-a-csv.md]: A list/slice.

**Pull request merged**